lib/collections/addon/components/collection-submission-confirmation-modal/template.hbs

```diff
@@ -0,0 +1,26 @@
+<BsModal 
+    @open={{@openModal}}
+    @onSubmit={{action @addToCollection}}
+    @onHidden={{action @cancel}}
+    as |modal|
+>
+    <modal.header>
+        <h3>{{t (concat this.i18nKeyPrefix 'header')}}</h3>
+    </modal.header>
+    <modal.body class=componentCssClassName>
```

Comment:
I'm guessing you copied `delete-node-modal`, which, apparently, has some cruft left over from when we were using `ember-component-css`:
```suggestion
    <modal.body>
```